File: xui_lib/src/main/java/com/stepyen/xui/TestActivity.java
package com.stepyen.xui;

import android.os.Bundle;

import android.widget.LinearLayout;


import com.stepyen.xui.utils.ResUtils;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

/**
 * date：2019/6/28
 * author：stepyen
 * description：
 */
public class TestActivity extends AppCompatActivity {
    protected LinearLayout mWelcomeLayout;
    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        initView();
    }

    private void initView() {
        mWelcomeLayout = new LinearLayout(this);
        LinearLayout.LayoutParams params = new LinearLayout.LayoutParams(LinearLayout.LayoutParams.MATCH_PARENT, LinearLayout.LayoutParams.MATCH_PARENT);
        mWelcomeLayout.setLayoutParams(params);
        mWelcomeLayout.setOrientation(LinearLayout.VERTICAL);
        mWelcomeLayout.setBackgroundColor(ResUtils.getColor(R.color.green));
        setContentView(mWelcomeLayout);
    }
}

File: app/src/main/java/com/stepyen/xuidemo/activity/TestActivity.kt
package com.stepyen.xuidemo.activity

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.stepyen.xui.utils.ResUtils
import com.stepyen.xui.utils.TextViewCountTimeHelp
import com.stepyen.xui.widget.spinner.MaterialSpinner
import com.stepyen.xuidemo.R
import com.stepyen.xutil.tip.ToastUtils
import kotlinx.android.synthetic.main.activity_test.*

/**
 * date：2022/12/5
 * author：stepyen
 * description：
 *
 */
class TestActivity  : AppCompatActivity() {

    var textViewCountTimeHelp: TextViewCountTimeHelp?=null


    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_test)

//        spinner1
//            .setItems<String>(*ResUtils.getStringArray(R.array.sort_mode_entry))
//        spinner1
//            .setArrowColor(ResUtils.getColor(R.color.xui_config_color_red))
//        spinner1
//            .setOnItemSelectedListener(object : MaterialSpinner.OnItemSelectedListener<Any?> {
//
//            override fun onItemSelected(
//                view: MaterialSpinner?,
//                position: Int,
//                id: Long,
//                item: Any?
//            ) {
//            }
//        })



        testTv.setOnClickListener {
            textViewCountTimeHelp = TextViewCountTimeHelp(testTv,5000,1000)
            textViewCountTimeHelp?.start()
        }

    }




}